### ISSUE TITLE
腾讯视频失效？

### ISSUE THREAD

**Issue by jiangjiangflies** [soimort/you-get#1022]
http://v.qq.com/cover/j/js93r5pmsm6d49n/a0019zbdqo3.html

**Comment by zhangn1985** [comment]
@soimort-bot Please test this link: http://v.qq.com/cover/j/js93r5pmsm6d49n/a0019zbdqo3.html

**Comment by soimort** [comment]
Wait, 

```
$ you-get -di http://v.qq.com/cover/j/js93r5pmsm6d49n/a0019zbdqo3.html 
you-get: version 0.4.367, a tiny downloader that scrapes the web.
you-get: ['http://v.qq.com/cover/j/js93r5pmsm6d49n/a0019zbdqo3.html']
Traceback (most recent call last):
  File "/home/soimort/Projects/you-get/you-get", line 11, in <module>
    you_get.main(repo_path=_filepath)
  File "/home/soimort/Projects/you-get/src/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/home/soimort/Projects/you-get/src/you_get/common.py", line 1274, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/home/soimort/Projects/you-get/src/you_get/common.py", line 1187, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/home/soimort/Projects/you-get/src/you_get/common.py", line 1030, in download_main
    download(url, **kwargs)
  File "/home/soimort/Projects/you-get/src/you_get/common.py", line 1267, in any_download
    m.download(url, **kwargs)
  File "/home/soimort/Projects/you-get/src/you_get/extractors/qq.py", line 33, in qq_download
    qq_download_by_vid(vid, title, output_dir, merge, info_only)
  File "/home/soimort/Projects/you-get/src/you_get/extractors/qq.py", line 14, in qq_download_by_vid
    _, ext, size = url_info(url, faker=True)
  File "/home/soimort/Projects/you-get/src/you_get/common.py", line 353, in url_info
    response = request.urlopen(request.Request(url, headers = fake_headers), None)
  File "/usr/lib/python3.5/urllib/request.py", line 162, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 471, in open
    response = meth(req, response)
  File "/usr/lib/python3.5/urllib/request.py", line 581, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.5/urllib/request.py", line 509, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 443, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 589, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 405: Not Allowed
```

@soimort-bot Test again: http://v.qq.com/cover/j/js93r5pmsm6d49n/a0019zbdqo3.html

**Comment by soimort** [comment]
Worked fine at first, and got a `405` error, and then `403`.

This is tricky.

**Comment by Ezio2** [comment]
all failed with 403.

http://v.qq.com/page/a/2/z/k01900esl3l.html

http://v.qq.com/cover/r/rhdpz0mvemcfbuo.html?vid=c0191w7w9eo

**Comment by luojinfang** [comment]
http://vv.video.qq.com/getvinfo
http://vv.video.qq.com/getvkey
These two interfaces

**Comment by soimort** [comment]
@soimort-bot Test this: http://v.qq.com/cover/w/wkfi7sdq44mto0i.html?vid=c0192ra25qz

**Comment by soimort** [comment]
Actually that's the wrong video.

@soimort-bot Try this: http://v.qq.com/cover/w/wkfi7sdq44mto0i/c0192ra25qz.html

**Comment by yuderbin** [comment]
I fix it in my pull request.
python3 you-get 'http://v.qq.com/cover/6/65d2otz0s4jhrrp.html?vid=l0020gwuwzx'
Site:       QQ.com
Title:      杨紫又变脸已认不出来 大鹏的女儿都这么大了？
Type:       MPEG-4 video (video/mp4)
Size:       9.08 MiB (9516011 Bytes)

Downloading 杨紫又变脸已认不出来 大鹏的女儿都这么大了？.mp4 ...
 100% (  9.1/  9.1MB) ├███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████┤[1/1]  232 kB/s

**Comment by soimort** [comment]
Fixed by #1062.
